spec/pages/attend.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+ RSpec.describe "pages/attend.html.erb",  type: :view do
+
+     it "Should render top banner section" do
+       render "shared/top_banner.html.erb"
```

Comment:
dont render other files than the one you're describing.

So: 

```ruby
render

expect (rendered).to have_content "Taste of Code"
```

Or find out how to test wether a view renders a partial, something among these lines:
```ruby
# this is pseudo code, see if you get it to work!
expect(render).to render_template(partial: 'shared/top_banner'
```